lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -32,8 +74,9 @@ module.exports = function({workspace, options}) {
 			});
 		})
 		.then((processedResources) => {
-			return Promise.all(processedResources.map((resource) => {
+			return Promise.all(processedResources.map(async (resource) => {
 				log.verbose("Writing flexibility changes bundle");
+				await updateFLdependency();
```

Comment:
Processors return an array of resources, which should all be written to the workspace, although (currently) in this case it always returns one resource.

But updating the manifest should only happen once, right? So I would expect it to be done after the `Promise.all`.